systers_portal/meetup/models.py

```diff
@@ -45,3 +45,13 @@ def clean_fields(self, *args, **kwargs):
         """Validate meetup date is not less than today's date"""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+
+
+class Rsvp(models.Model):
+    """ Users RSVP for particular meetup """
+    user_id = models.ForeignKey(SystersUser)
+    meetup_id = models.ForeignKey(Meetup)
+    is_coming = models.BooleanField()
```

Comment:
Then those extra 10 people maybe should register themselves and rsvp.
It's just a common thing to let people have +1 (at least in European/American cultures). I assume it began as inviting your spouse/SO. Check the meetup.com. Plus we have it in the requirements doc (I think).